Runtime/Editor/LootLockerExtensionCalls.cs

```diff
@@ -0,0 +1,143 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.NetworkInformation;
+using LLlibs.ZeroDepJson;
+using LootLocker.Extension.Requests;
+using LootLocker.Requests;
+using UnityEditor.VersionControl;
+using UnityEngine;
+
+namespace LootLocker.Admin
+{
+    public partial class LootLockerAdminManager
+    {
+                
+        public static void AdminLogin(string email, string password, Action<LoginResponse> onComplete)
+        {
+            if (string.IsNullOrEmpty(email) || string.IsNullOrEmpty(password))
+            {
+                Debug.Log("Input is invalid!");
+                return;
+            }
+
+            AdminLoginRequest request = new AdminLoginRequest();
+            request.email = email;
+            request.password = password;
+
+            AdminLogin(request, onComplete);
+        }
+
+        public static void AdminLogin(AdminLoginRequest data, Action<LoginResponse> onComplete)
```

Comment:
This goes for all the methods, but I don't get why we'd do an overload for all these methods that take the contstructed objects? I don't think we need it, just move this method into the former.

Reply:
Removed ur right!